MAUI/DataGrid/Column-types.md

```diff
@@ -908,6 +908,30 @@ dataGrid.Columns.Add(numericColumn);
 {% endhighlight %}
 {% endtabs %}
 
+### Show row header
+
+The `SfDataGrid` allows you to add the content inside the row header using the RowHeaderTemplate. It allows you to load any view inside the row header.
```

Comment:
The row header is a type of column that is placed as the first cell of each row and remains frozen. To enable the row header, set `SfDataGrid.ShowRowHeader` to `true`.

Additionally, the `SfDataGrid` allows you to customize the row header width using the `SfDataGrid.RowHeaderWidth` property. The default value is 25.